src/nvim/main.c

```diff
@@ -1494,9 +1498,18 @@ static void init_params(mparm_T *paramp, int argc, char **argv)
 /// Initialize global startuptime file if "--startuptime" passed as an argument.
 static void init_startuptime(mparm_T *paramp)
 {
+  // check if we are currently running as the tui or the main process
+  // the main process is always called with the --embed flag
```

Comment:
I think this comment can just be deleted. It's quite obvious what the code does.

Reply:
done